src/FSharpPlus/Extensions/Map.fs

```diff
@@ -56,7 +56,7 @@ module Map =
             match f v with
             | Some v -> yield (k, v)
             | None    -> () }
-
+    
```

Comment:
Can you remove this unintentional line change?

Reply:
Removed